jackson-jq/src/main/java/net/thisptr/jackson/jq/Scope.java

```diff
@@ -48,25 +49,25 @@ private Map<String, String> debugFunctions() {
 	private ModuleLoader moduleLoader;
 
 	public interface ValueWithPath {
-		JsonNode value();
```

Comment:
This is part of the public API, and this change would break e.g. `JsonNode value = scope.getValueWithPath(...).value()`

I think we can avoid the breaking change by calling Supplier#get() inside ValueWithPathImpl#value(). Wdyt?

Reply:
@eiiches Good catch! 
Changed to maintains backward compatibility. thanks a lot!